src/main/java/org/openmrs/uitestframework/page/Page.java

```diff
@@ -196,6 +198,11 @@ private void setTextNoEnter(WebElement element, String text) {
 	public void clickOn(By by) {
 		findElement(by).click();
 	}
+	
+	public void clickOnLast(By by) {
```

Comment:
this method exists already, did you check with the qa branch?

Reply:
Yeah this exists only on qa branch  not on master branch thats why it cant be or imported by clinicianPatientPage in distro, ,i saw that all methods that introduced currently have impact on classes that we inherit especially from master page class

Reply:
The challenge is importing it fron qa , when you checkout from master branch, this method is not there which is the limitation

Reply:
Hello @k-joseph  any suggestion about this, You may be help us to understand it more

Reply:
Hello @k-joseph , this commit wont pass , i have tried different ways running on qa branch but it failed because master branch doesnt have this method, shall we first merge this commit and we continue to test it out please

Reply:
> Yeah this exists only on qa branch not on master branch thats why it cant be or imported by clinicianPatientPage in distro, ,i saw that all methods that introduced currently have impact on classes that we inherit especially from master page class

Just as @k-joseph said, this method exists from the base class(Page) present in the openmrs-contrib-uitestframework. Take a look at it [here](https://github.com/openmrs/openmrs-contrib-uitestframework/blob/6fe16add5570b71024a1025f418c839a8f2cbae4/src/main/java/org/openmrs/uitestframework/page/Page.java#L201)